File: server/src/main/java/ak/eep/web/server/server/Room.java
package ak.eep.web.server.server;

public class Room {
    public static final String EEP_COMMAND = "[EEPCommand]";
    public static final String LOG = "[Log]";
    public static final String PING = "[Ping]";
    public static final String ROOM = "[Room]";
    public static final String AVAILABLE_DATA_TYPES = "[AvailableDataTypes]";

    public static String ofDataType(String dataType) {
        return "[Data-" + dataType + "]";
    }

    public static String dataTypeOf(String room) {
        if (room.startsWith("[Data-") && room.endsWith("]")) {
            return room.substring(6, room.length() - 1);
        }
        throw new IllegalStateException("No data room: " + room);
    }
}

File: node-server/src/server/room.ts
export default class Room {
  static EEP_COMMAND = '[EEPCommand]';
  static LOG = '[Log]';
  static PING = '[Ping]';
  static ROOM = '[Room]';
  static URLS = '[URLs]';
  static SERVER_SETTINGS = '[Server Settings]';
  static AVAILABLE_DATA_TYPES = '[AvailableDataTypes]';

  public static ofDataType(dataType: string): string {
    return '[Data-' + dataType + ']';
  }
}
